akka-projection-testkit/src/main/scala/akka/projection/testkit/ProjectionTestKit.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.testkit
+
+import akka.Done
+import akka.actor.testkit.typed.TestKitSettings
+import akka.actor.testkit.typed.scaladsl.{ ActorTestKit, _ }
+import akka.annotation.ApiMayChange
+import akka.projection.Projection
+
+import scala.concurrent.duration._
+import scala.concurrent.{ Future, Promise }
+@ApiMayChange
+object ProjectionTestKit {
+  def apply(testKit: ActorTestKit): ProjectionTestKit =
+    new ProjectionTestKit(testKit)
+}
+
+@ApiMayChange
+final class ProjectionTestKit private[akka] (testKit: ActorTestKit) {
+
+  private implicit val system = testKit.system
+  private implicit val settings: TestKitSettings = TestKitSettings(system)
+
+  def run(proj: Projection)(testFunc: => Unit): Future[Done] =
+    runInternal(proj, testFunc, settings.SingleExpectDefaultTimeout.dilated, 100.millis)
+
+  def run(proj: Projection, max: FiniteDuration)(testFunc: => Unit): Future[Done] = {
+    runInternal(proj, testFunc, max, 100.millis)
+  }
+
+  def run(proj: Projection, max: FiniteDuration, interval: FiniteDuration = 100.millis)(
```

Comment:
interval not used, and should probably not have a default value if we have the other overload without the interval